star-proto/scripts/check_nanopb_sync.sh

```
@@ -0,0 +1,99 @@
+    matches=$(awk -v field="${field}" '$1 == field' "${file}")
+    local match_count
+    match_count=$(echo -n "${matches}" | grep -c '^' || true)
+    if [[ "${match_count}" -gt 1 ]]; then
+        echo "ERROR: ${field} defined ${match_count} times in ${file}" >&2
+        return 1
+    fi
+    echo "${matches}" \
+        | grep -oE 'max_count:[0-9]+' \
+        | grep -oE '[0-9]+' \
+        || true
+}
```

Shellcheck (0.11.0)

[info] This function is never invoked. Check usage (or ignored if invoked indirectly).

(SC2329)